include/graphblas/algorithms/amg/amg.hpp

```diff
@@ -0,0 +1,230 @@
+
+/*
+ *   Copyright 2021 Huawei Technologies Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file amg.hpp
+ * @author Alberto Scolari (alberto.scolari@huawei.com)
+ * @brief File with the main routine to run a full AMG simulation, comprising multi-grid runs
```

Comment:
The first line of a doxygen comment is always its brief description, making `@brief` unnecessary (please remove:)

Also the brief description is a bit long;) The comma can probably be an end-of-sentence, with the remainder starting on a new paragraph?